docs/UserGuide.adoc

```diff
@@ -29,174 +29,452 @@ apps.
 == Quick Start
 
 1.  Ensure you have JRE 9 or later installed.
-2.  Download the `.jar`.
+2.  Download the latest `.jar` file link:{repoURL}/releases[here].
 3.  Place it in the folder where you want the data files to be stored.
-4.  Type in a command and press `Enter` to execute it. For a start, type
-`help` and press `Enter` to see the help text.
+4.  Double-click the `.jar` file to start the app. The GUI should appear
+in a few seconds
++
+image::Ui.png[width="800"]
++
+5.  Type in a command and press kbd:[Enter] to execute it. For a start, type
+`help` and press kbd:[Enter] to see the help page.
+6.  Some example commands you can try:
+* `contacts list` : lists all contacts
+* `contacts add n/John Doe p/98765432 e/johnd@example.com a/311, Clementi Ave 2, #02-25 t/friends t/owesMoney`: adds contact named `John Doe` to Cow
+* `tasks list` : lists all tasks
+* `tasks delete 2` : deletes the 2nd task shown in the list
+* `exit` : exits Cow
+
+[WARNING]
+Ensure that your screen resolution is at least 1200x800, or the calendar
+might not show properly.
 
 == Features
 
-**Command Format**
+====
+*Command Format*
 
 * Words in `UPPER_CASE` are the parameters to be supplied by the user
-e.g. in `add n/NAME`, `NAME` is a parameter which can be used as
-`add n/John Doe`.
+e.g. in `contacts add n/NAME`, `NAME` is a parameter which can be used as
+`contacts add n/John Doe`.
 * Items in square brackets are optional e.g `n/NAME [t/TAG]` can be used
 as `n/John Doe t/friend` or as `n/John Doe`.
 * Items with `…` after them can be used multiple times including zero
 times e.g. `[t/TAG]...` can be used as
-`\` (i.e. 0 times),`t/friend`,`t/friend t/family` etc.
+`\` (i.e. 0 times), `t/friend`, `t/friend t/family` etc.
 * Parameters can be in any order e.g. if the command specifies
 `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
+====
 
 [TIP]
 When `INDEX` is a parameter, it refers to the index of the item shown on the task or person list
 
+[NOTE]
+You can add multiple people or multiple tasks with the same name because internally we track them with
+a unique id.
```

Comment:
I don't think we should talk about our implementation as this is targeted at users. We should still somehow justify why we're allowing "duplicate" people and tasks. Something like "Don't worry about having friends or tasks with the same name -- just add them and Cow will process them correctly."?

Reply:
Great! I was looking for a better way to phrase this